File: Stephannie/Exercise-9.py
#BMI interpretation

weight = float(input("Enter your current weight in kg: "))

height = float(input("Enter your curent height in m: "))

BMI_original = weight / height ** 2
BMI = round(BMI_original, 3)

if BMI <= 18.5:
   print(f"Your BMI is {BMI}, you are underweight.")
elif BMI >= 18.5 and BMI <= 24.9:
   print (f"Your BMI is {BMI}, you have a normal weight.")
elif BMI >= 25 and BMI <= 29.9:
   print(f"Your BMI is {BMI}, you are overweight.")
elif BMI >= 30 and BMI <= 34.9:
   print(f"Your BMI is {BMI}, you are Obese.")
elif BMI >= 35:
   print(f"Your BMI is {BMI}, you are clinically Obese.")
else:
   print("Invalid input for weight and height.Try again.")   
   

#Done

File: YaroWilliams/Exercise9.java
import java.util.Scanner;
public class Exercise9 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("Enter your weight: " );
        double weight = input.nextDouble();

        System.out.println("Enter your height: ");
        double height = input.nextDouble();

        int BMI = (int) (weight / (height * height));

        String bodyMass =" ";

        if(BMI <18.5){
            bodyMass="Underweight";
        }else if(BMI >=18.5 && BMI <=24.9){
            bodyMass="Normal weight";
        }else if(BMI >=25 && BMI <=29.9){
            bodyMass="Overweight";
        }else if(BMI > 35){
            bodyMass=" Clinically Obese";
        }    

        System.out.println("Your BMI is: " + BMI+ ", you are " + bodyMass);
        input.close();
        
    }
}
